subjects/doop.en.md

```diff
@@ -42,7 +42,7 @@ No Modulo by 0$
 student@ubuntu:~/piscine-go/test$ ./doop 9223372036854775807 + 1
 Overflow
 student@ubuntu:~/piscine-go/test$ ./doop -9223372036854775809 "*" 3
-Overflow
+0
```

Comment:
Why ? -9223372036854775809 * 3 appear as an overflow to me but maybe I'm wrong

Reply:
-9223372036854775809 * 3 this is already an overflow so i changed the calculation to subtract instead of multiply

Reply:
Ok, I'm noticing another error : `-9223372036854775808` is the smallest `int64`, not `-9223372036854775809` (invalid)

Reply:
its to show that if one of the arguments is out of range by the atoi it will print "0" because its an overflow

Reply:
In that case even `-9223372036854775809 + 0` will fail, because Atoi will trigger an error since `-9223372036854775809` is an invalid number (like `123abc` for example).
It is not an integer overflow.

Reply:
yes, and in both cases the output will be 0
that case it's just to show that if you put a number that is too big(invalid) the program prints "0"